packages/blockchain-link/src/workers/solana/index.ts

```diff
@@ -3,15 +3,69 @@ import type * as MessageTypes from '@trezor/blockchain-link-types/lib/messages';
 import { CustomError } from '@trezor/blockchain-link-types/lib/constants/errors';
 import { BaseWorker, ContextType, CONTEXT } from '../baseWorker';
 import { MESSAGES, RESPONSES } from '@trezor/blockchain-link-types/lib/constants';
-import { Connection, PublicKey } from '@solana/web3.js';
+import { Connection, ParsedTransactionWithMeta, PublicKey } from '@solana/web3.js';
+import { Transaction } from '../..';
```

Comment:
nit: I'd rather see import from blockchain-link-types rather than from blockchain-link/index here

Reply:
fixed in https://github.com/trezor/trezor-suite/pull/9498/commits/a21e445e80cf8ffe887af86b366181efb4040ad5